pages/tutorials/3.0/getting-started/code-blocks.md

```diff
@@ -7,20 +7,29 @@
+This tutorial is the first one you should read if you're using SFML with the Code::Blocks IDE, and the GCC compiler (this is the default one).
+It will explain how to configure your SFML projects.
 
 ## Installing SFML
 
 First, you must download the SFML SDK from the [download page](https://www.sfml-dev.org/download.php "Go to the download page").
 
-There are multiple variants of GCC for Windows, which are incompatible with each other (different exception management, threading model, etc.). Make sure you select the package which corresponds to the version that you use. If you are unsure, check which of the libgcc_s_sjlj-1.dll or libgcc_s_dw2-1.dll files is present in your MinGW/bin folder. If MinGW was installed along with Code::Blocks, you probably have an SJLJ version.  
+There are multiple variants of GCC for Windows, which are incompatible with each other (different exception management, threading model, etc.).
+Make sure you select the package which corresponds to the version that you use.
+If you are unsure, check which of the libgcc_s_sjlj-1.dll or libgcc_s_dw2-1.dll files is present in your MinGW/bin folder.
+If MinGW was installed along with Code::Blocks, you probably have an SJLJ version.
+
 If you feel like your version of GCC can't work with the precompiled SFML libraries, don't hesitate to [build SFML yourself](https://www.sfml-dev.org/tutorials/2.6/compile-with-cmake.php "How to compile SFML"), it's not complicated.
 
-You can then unpack the SFML archive wherever you like. Copying headers and libraries to your installation of MinGW is not recommended, it's better to keep libraries in their own separate location, especially if you intend to use several versions of the same library, or several compilers.
+You can then unpack the SFML archive wherever you like.
+Copying headers and libraries to your installation of MinGW is not recommended, it's better to keep libraries in their own separate location, especially if you intend to use several versions of the same library, or several compilers.
 
 ## Creating and configuring a SFML project
 
-The first thing to do is choose what kind of project to create. Code::Blocks offers a wide variety of project types, including an "SFML project". **Don't use it!** It hasn't been updated in a long time and is likely incompatible with recent versions of SFML. Instead, create an Empty project. If you want to get rid of the console, in the project properties, go to the "Build targets" tab and select "GUI application" in the combo box instead of "Console application".
+The first thing to do is choose what kind of project to create.
+Code::Blocks offers a wide variety of project types, including an "SFML project". **Don't use it!** It hasn't been updated in a long time and is likely incompatible with recent versions of SFML.
```

Comment:
Missing line break

Reply:
Turns out that the character prior to `**Don't` is a non-ASCII whitespace character of some kind. It didn't match the regular expression I was using to detect multiple sentences in the same line of code. I went through and removed all the Unicode characters from these files so they all report as ASCII.